generic3g/ComponentSpecParser/parse_var_specs.F90

```diff
@@ -24,21 +24,21 @@ module function parse_var_specs(hconfig, timestep, refTime, rc) result(var_specs
 
       subcfg = ESMF_HConfigCreateAt(hconfig,keyString=COMPONENT_STATES_SECTION, _RC)
 
-      call parse_state_specs(var_specs, subcfg, COMPONENT_INTERNAL_STATE_SECTION,  timestep, refTime, _RC)
-      call parse_state_specs(var_specs, subcfg, COMPONENT_EXPORT_STATE_SECTION, timestep, refTime, _RC)
-      call parse_state_specs(var_specs, subcfg, COMPONENT_IMPORT_STATE_SECTION, timestep, refTime, _RC)
+      call parse_state_specs(var_specs, subcfg, COMPONENT_INTERNAL_STATE_SECTION,  timestep, refTime_offset, _RC)
+      call parse_state_specs(var_specs, subcfg, COMPONENT_EXPORT_STATE_SECTION, timestep, refTime_offset, _RC)
+      call parse_state_specs(var_specs, subcfg, COMPONENT_IMPORT_STATE_SECTION, timestep, refTime_offset, _RC)
```

Comment:
I think var specs will need an absolute time.   offset works well for parent-child relationship, but two different varspecs from different components may be offset relative to entirely unrelated times.